google/cloud/storage/tests/grpc_integration_test.cc

```diff
@@ -383,6 +384,14 @@ TEST_P(GrpcIntegrationTest, NativeIamCRUD) {
   EXPECT_EQ(get->version(), set->version());
   EXPECT_EQ(get->etag(), set->etag());
 
+  auto test = client->TestBucketIamPermissions(
+      bucket_name, {"storage.buckets.get", "storage.buckets.setIamPolicy",
+                    "storage.objects.update"});
+  ASSERT_STATUS_OK(test);
+  EXPECT_THAT(*test,
+              ElementsAre("storage.buckets.get", "storage.buckets.setIamPolicy",
```

Comment:
nit: if we do not care about the order (and I think we do not), maybe `UnorderedElementsAre(...)`